### PR TITLE
Join ovnkube-controller and ovnkube-node container for multizone setup 

### PR DESCRIPTION
**Pull request by npinaeva**
TOOD: Remove ovn-k image tweak commit when https://github.com/openshift/ovn-kubernetes/pull/1833 is merged

### CODE REVIEW

**Review by trozet — COMMENTED** [review]
looks good, lets see how CI goes

File: bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml [hunk 1]
```diff
@@ -960,9 +813,15 @@ spec:
...
             multi_network_policy_enabled_flag="--enable-multi-networkpolicy"
           fi
+          
+          admin_network_policy_enabled_flag=
+          if [[ "{{.OVN_ADMIN_NETWORK_POLICY_ENABLE}}" == "true" ]]; then
+            admin_network_policy_enabled_flag="--enable-admin-network-policy"
+          fi
 
-          exec /usr/bin/ovnkube --init-node "${K8S_NODE}" \
+          exec /usr/bin/ovnkube --init-ovnkube-controller-with-node "${K8S_NODE}" \
```
Comment (jcaamano) [inline bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml:822]: Didn't we have separate flags for this? Or are you using an old image?

File: bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml [hunk 2]
```diff
@@ -698,155 +698,8 @@ spec:
...
-            fieldRef:
-              fieldPath: spec.nodeName
-        ports:
-        - name: metrics-port-c  # c stands for controller, we only have 15 characters max for ports on a windows platform
-          containerPort: 29113
-        terminationMessagePolicy: FallbackToLogsOnError
-
       # ovnkube-node: does node-level bookkeeping and configuration
-      - name: ovnkube-node
+      - name: ovnkube-controller
```
Comment (jcaamano) [inline bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml:702]: perhaps tweak the comment above

File: bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml (hunk 1 above)
Comment (npinaeva) [reply bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml:822]: updating 2 files is too difficult :D thanks a lot for catching these, I will update the PR when the first CI round ends just to see the results.

**Pull request closed without merging**